modules/admin_manual/pages/qnap/index.adoc

```diff
@@ -117,18 +117,28 @@ Create an entry like in the following example with the correct IP address:
 <?php
 CONFIG = array(
     'overwriteprotocol' => 'https',
-    'overwritehost' => '203.0.113.0',
-    'overwrite.cli.url' => 'https://203.0.113.0/owncloud/',
+    'overwritehost' => '203.0.113.0:4490',
+    'overwrite.cli.url' => 'https://203.0.113.0:4490/',
 );
 ----
 
-ownCloud on QNAP is set up as a subdirectory by default.
-
```

Comment:
Question, what is the reason going from subdir to port because this is about assigning a static IP where the default is a port? When you go for a reverse proxy, using port is correct, but is covered in the topic below. Pls explain.

Reply:
When I installed the current version of OwnCloud on my QNAP, it was exposed on port 4490 without a subdir. I don't know if this is how it's supposed to be. I preferred fixing the above for consistency. Let met know if that's the case, otherwise happy to revert it and keep the scope of this PR to reverse proxy.